### ISSUE TITLE
Dark Theme

### ISSUE THREAD

**Issue by B0gos** [B0gos/lab-agile-planning#5]
**As a** User
**I need**  a togglable dark mode / theme for the application
**So that** I can use this application comfortable in low light conditions

### Details and Assumptions
* Darkmode is a white font on a dark background
### Acceptance Criteria
```gherkin
Given the dark mode is enabled
When the page loads
Then the text should be readable but the background dark
```
